examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -194,19 +189,20 @@ def pad_fn(image, boxes):
     classification_loss="focal",
     box_loss="smoothl1",
     optimizer=optimizer,
+    metrics=[],
 )
 
+
 callbacks = [
     keras.callbacks.TensorBoard(log_dir="logs"),
-    keras.callbacks.ReduceLROnPlateau(patience=5),
-    keras.callbacks.EarlyStopping(patience=10),
+    keras.callbacks.ReduceLROnPlateau(monitor="loss", patience=5),
+    keras.callbacks.EarlyStopping(monitor="loss", patience=10),
     keras.callbacks.ModelCheckpoint(CHECKPOINT_PATH, save_weights_only=True),
-    PyCOCOCallback(eval_ds, "xywh"),
+    keras_cv.callbacks.PyCOCOCallback(eval_ds, "xywh"),
 ]
 
 history = model.fit(
     train_ds,
-    validation_data=eval_ds,
-    epochs=35,
+    epochs=50,
```

Comment:
Is change of epoch count intentional?

Reply:
yeah this is intentional, sometimes it takes this long to converge